docs/examples/dodge-rock.md

```diff
@@ -0,0 +1,37 @@
+# Dodge the Rocks
```

Comment:
This one is slightly too hard, I'd say reduce the number of rocks by ~15-20% cause right now it's hard to stay alive more than 5 seconds :P